cves/CVE-2014-0483.yml

```diff
@@ -28,11 +28,11 @@ announced_instructions: |
   source for this is Chrome's Stable Release Channel
   (https://chromereleases.googleblog.com/).
   Please enter your date in YYYY-MM-DD format.
-announced_date:
+announced_date: 2014-08-20
 published_instructions: |
   Is there a published fix or patch date for this vulnerability?
   Please enter your date in YYYY-MM-DD format.
-published_date:
+published_date: 2014-08-20
```

Comment:
When Researching my Django vulnerabilities, I noticed that they tend to publish the release before the disclosure. I'd double check the date.

Reply:
The four releases related to this vulnerability, Django 1.4.14, Django 1.5.9, Django 1.6.6, and Django 1.7 release candidate 3, are tagged August 20th, 2014, as can be seen here: https://github.com/django/django/releases?after=1.6.8